CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ else()
     endif()
 endif()
 
+set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "")
```

Comment:
Would this remain same on windows ?

Reply:
Else we can check if it is compiling for windows and set a different default path

Reply:
Users could use that path on windows, but I assume they most likely would set it because there is no standard install location.

Reply:
Previously, we set the path to `/opt/rocm/x86_64-w64-mingw32` on windows, which is mainly for cross-compiling with mingw.